superset/utils/oauth2.py

```diff
@@ -192,3 +192,8 @@ class OAuth2ClientConfigSchema(Schema):
     )
     authorization_request_uri = fields.String(required=True)
     token_request_uri = fields.String(required=True)
+    request_content_type = fields.String(
```

Comment:
I might be mistaken, but I believe this validation is only applied to the client_info contained within `encrypted_extra` when provided by a user? Is the intent that this is where the `request_content_type` is going to be provided, or is it going to be set as a default for the Trino engine spec?

Reply:
Right, I think the idea is that DB engine specs can have a default content type (defined in the `oauth2_token_request_type` class attribute), but it can be overridden on a per-database basis by setting it in the `encrypted_extra`.